_posts/2019-10-28-security-update.markdown

```diff
@@ -0,0 +1,28 @@
+---
+title: "Security Update"
+layout: post
+author: jjergus
+category: blog
+---
+
+A security update has been released for all supported HHVM versions. Please
+update to one of the following versions to make sure you're secure:
+
+- 4.28.2
+- 4.27.1
+- 4.26.1
+- 4.25.1
+- 4.24.1
+- 4.23.2
+- 4.8.6
+- 3.30.12
+
+This security update addresses multiple potential vulnerabilies, mostly found by
+running a new fuzzer over the HHVM codebase.
+
+More information can be found in the respective CVEs:
```

Comment:
well, this isn't really true as the CVEs are empty :p

Reply:
will be true eventually :)